.circleci/conditional_config.yml

```diff
@@ -512,6 +499,9 @@ workflows:
     jobs:
       - pipeline_settings
       - checkout_code
+      - migrations_tests:
```

Comment:
That's a rather big hammer, running all the migration tests. How about just moving out `bundle exec database_consistency...`